sources/ham/tests/ham-test-thrift/_build.ham

```diff
@@ -1,6 +1,10 @@
 SubDir TOP ;
 
-ushScripts ham-test-thrift ;
+local DIR = [ FDirName $(TOP) $(SUBDIR) ] ;
+# ECHO "... DIR:" $(DIR) ;
+local SH_FILES = [ FGristFiles [ lffGlob $(DIR) : .sh ] ] ;
```

Comment:
We probably want to move FGristFiles in lffGlob, I dont think there's any reason to glob with doing the Grist part. --- Ideally we'd follow whatever is the convention/done in the current .ham rules.

Btw, for reference:
- https://swarm.workshop.perforce.com/view/guest/perforce_software/jam/src/Jam.html
- https://swarm.workshop.perforce.com/view/guest/dick_dunbar/jam/src/Jambase.html

Those explain what Grist means. The short of it is that its tagging the filename with the folder name to make sure that two files with the same name in different folders are always  distinguished.